src/sagemaker/pipeline.py

```diff
@@ -221,6 +227,19 @@ def deploy(
             return predictor
         return None
 
+    @runnable_by_pipeline
+    def create(self, instance_type: str):
+        """Create a SageMaker Model Entity
+
+        Args:
+            instance_type (str): The EC2 instance type that this Model will be
+                used for, this is only used to determine if the image needs GPU
+                support or not.
+        """
+        if isinstance(self.sagemaker_session, PipelineSession):
+            self.sagemaker_session.init_model_step_arguments(self)
```

Comment:
We can move this to the runnable_by_pipeline decorator. The less change we make here, the better.

Reply:
Good point! and it reduces duplicate code too. Will move